renku-core-client/src/main/resources/application.conf

```diff
@@ -0,0 +1,10 @@
+
+services {
+
+  renku-core-latest {
+    url = "http://localhost:80"
+    url = ${?RENKU_CORE_LATEST_URL}
+  }
+
+  renku-core-service-urls = ${?RENKU_CORE_SERVICE_URLS}
```

Comment:
There is no sensible default possible, right? Was just thinking how the `RenkuCoreUri#ForSchema` is working - i.e. which kind of value is expected - an array?

Reply:
I see your point. Unfortunately, I'm not sure if we can default to something as we expect the schema version to be encoded into the hostname. I mean, we could default to, let's say `http://renku-core-v10`, but in case you'd like to run it successfully, a relevant entry in the `/etc/hosts` would need to be added. Maybe I'll add such a default.

Reply:
Ok, I see, not easy - then it's good to leave it. So an entry in `/etc/hosts` is required? Or "just" a resolvable url? (from the pov of the module)